src/bundle/Resources/views/ui/layout.html.twig

```diff
@@ -111,7 +111,7 @@
         <div id="react-udw"></div>
 
         {% block navigation %}
-            {% include '@ezdesign/part/navigation.html.twig' %}
+            {% include '@ezdesign/ui/menu/top_navigation.html.twig' %}
```

Comment:
We do not have any other navigation template, also twig block is called `navigation` so I think that this template name could stay as is, or we should change block name too. `menu` in combination with `navigation` looks weird.